### PR TITLE
Add atomic energy normalizer

### PR DESCRIPTION
**Pull request by maximilianvz**

### CODE REVIEW

File: newtonnet/models/newtonnet.py [hunk 1]
```diff
@@ -220,14 +220,15 @@ def forward(self, data):
...
         Ei = Ei * AM[..., None]  # (B,A,1)
-        if self.normalizer[0].size > 1:
-            Ei = self.inverse_normalize(Ei)
-            if self.aggregration == 'sum':
-                E = torch.sum(Ei, 1)
-            elif self.aggregration == 'mean':
-                E = torch.mean(Ei, 1)
-            elif self.aggregration == 'max':
-                E = torch.max(Ei, 1).values
+        if hasattr(self, "normalizer"):
```
Comment (leila-pujal) [inline newtonnet/models/newtonnet.py:223]: Why do you have to check now if the normalizer has this attribute?

Comment (maximilianvz) [reply newtonnet/models/newtonnet.py:223]: Now, when you run training (i.e., after you merge this PR), the `self` object will have an attribute, `normalizer`, because, in this PR, I added a line that does `self.normalizer = normalizer`. However, before this PR existed, when I ran trainings, this `normalizer` attribute was never instantiated, so trying to load the `best_model` will raise an error. The NewtonNet object saved in those earlier trainings doesn't have a `normalizer` attribute, so I had to add this check when checking MAEs on those earlier trainings.

Comment (leila-pujal) [inline newtonnet/models/newtonnet.py:223]: Why do you have to check now if the normalizer has this attribute?

**Review by leila-pujal — CHANGES_REQUESTED** [review]

File: newtonnet/models/newtonnet.py (hunk 1 above)
Comment (leila-pujal) [reply newtonnet/models/newtonnet.py:223]: I will answer your answer in the comment above here to resolve the review (first time reviewing, I got comment/review mess up).  I understand now the code flow thanks for your clarification. This looks good, I will check some other things before merging. I will let you know if I have more questions.
